Back End/MemorizeWords/MemorizeWords/Infrastructure/Persistance/Repository/WordRepository.cs

```diff
@@ -129,17 +129,31 @@ private static void ValidationAddUpdateWord(WordAddRequest wordAddRequest)
 
         public async Task<List<QuestionWordResponse>> GetQuestionWordsAsync()
         {
-            var randomWords = await Queryable().Where(x => !x.IsLearned)
+            int randomNumber = new Random().Next(0, 21);
+
+            var randomWordsPLtoTR = await Queryable().Where(x => !x.IsLearned)
                                    .OrderBy(x => Guid.NewGuid())
-                                   .Take(20)
+                                   .Take(randomNumber)
                                    .Select(x => new QuestionWordResponse()
                                    {
                                        Word = x.Word,
                                        Id = x.Id,
                                        WritingInLanguage = x.WritingInLanguage,
                                        Meaning = x.Meaning
                                    }).ToListAsync();
+            
+            var randomWordsTRtoPL = await Queryable().Where(x => !x.IsLearned)
+                .OrderBy(x => Guid.NewGuid())
+                .Take(20-randomNumber)
```

Comment:
take "20-randomNumber" to a variable

Reply:
randomWordsTRtoPL 

TR to PL olmasın adı. İleride kullanicilar kendi dilinden baska dile kullanacak. Genel bir isim vermeyi dene

Reply:
Yapıldı.